pulp_rpm/tests/functional/constants.py

```diff
@@ -53,6 +53,12 @@
 "content_summary" field on "../repositories/../versions/../".
 """
 
+RPM_LARGE_EPEL_FIXTURE_URL = 'https://dl.fedoraproject.org/pub/epel/7Server/x86_64/'
```

Comment:
Test looks fine to me but I'm not sure if we should call this a "fixture".  Maybe just call it RPM_EPEL_FIXTURE_URL and name the test EpelSyncTestCase?  If the EPEL repo changes in such a way that it breaks Pulp, that might make it easier to see immediately what's going on, because the breakage probably wouldn't be due to the size of the repo.

Reply:
@dralley, I will work on those changes.